firmware/arduino/chessboard/motorDriver.ino

```diff
@@ -172,6 +172,9 @@ uint8_t moveStraight(uint8_t motor[], int endCol, int endRow)
   int *currentMotorPos;
   int startCol, startRow;
 
+  // Assigned at the end of each movement function to be checked for error handling
+  uint8_t statusCodeResult;
```

Comment:
The comment above and this declaration need to be done in `moveDirect` and `moveAlongEdges` in `movementFunctions.ino`

Reply:
This is based on where we're calling the primitives, which I explain in the next comment.